sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -36,52 +36,58 @@ testPy38Cython.mustRunAfter testPython38, testPy38CloudCoverage
 
 // TODO(BEAM-12000): Move tasks that aren't specific to 3.8 to Py 3.9.
 def posargs = project.findProperty("posargs") ?: ""
-def runDataframesTests = (posargs == "" || posargs.contains("dataframes"))
-logger.info('Running dataframes tests: ' + runDataframesTests)
+def runDataframeTests = (posargs == "" || posargs.contains("dataframe"))
+logger.info('Running dataframes tests: ' + runDataframeTests)
+
+// PyCoverage Precommit runs test suites that evaluate test coverage and compatibility of
+// particular dependencies. It is exercised on a single Python version.
+project.tasks.register("preCommitPyCoverage") {
+      dependsOn = ["testPy38CloudCoverage"]
+}
 
 // Create a test task for each major version of pyarrow
 toxTask "testPy38pyarrow-0", "py38-pyarrow-0", "${posargs}"
 test.dependsOn "testPy38pyarrow-0"
-preCommitPy38.dependsOn "testPy38pyarrow-0"
+preCommitPyCoverage.dependsOn "testPy38pyarrow-0"
```

Comment:
This is the right move for these tests, we should probably actually do the same thing for the dataframes tests so that they get counted towards codecoverage.

I'm approving this PR, feel free to merge as is or make that change. If you merge as is, I'll put up a follow up PR to move the dataframes tests into the coverage job.

Reply:
thanks, moving dataframe compatibility tests into preCommitPyCoverage